src/main/java/fi/helsinki/btls/App.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) {
             try {
                 Observation obs = gson.fromJson(s, Observation.class);
                 addObservation(obs);
-            } catch (Exception ex) {
+            } catch (Exception ignored) {
```

Comment:
I think it would be good to log exceptions here, for better debugging.

Reply:
If im reading this right, it will silently ignore observation data with invalid schema?